docs/_build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
+# This file hashes the configuration used when building these files. When it is not found, a full rebuild will be done.
```

Comment:
Should these files have been pushed up?

Reply:
No. It looks like someone built the sphinx docs locally and then accidentally committed those.